src/AccessibilityInsights.Win32/NativeMethods.cs

```diff
@@ -11,6 +12,17 @@ namespace AccessibilityInsights.Win32
     /// </summary>
     internal static partial class NativeMethods
     {
+        [DllImport("gdi32.dll", CharSet = CharSet.Auto, SetLastError = true, ExactSpelling = true)]
+        internal static extern uint GetDeviceCaps(IntPtr hDC, int nIndex);
+
+        //https://msdn.microsoft.com/en-us/library/windows/desktop/dd145062(v=vs.85).aspx
+        [DllImport("User32.dll")]
+        internal static extern IntPtr MonitorFromPoint([In]Point pt, [In]uint dwFlags);
+
+        //https://msdn.microsoft.com/en-us/library/windows/desktop/dn280510(v=vs.85).aspx
+        [DllImport("Shcore.dll")]
+        internal static extern IntPtr GetDpiForMonitor([In]IntPtr hmonitor, [In]DpiType dpiType, [Out]out uint dpiX, [Out]out uint dpiY);
```

Comment:
The underlying `GetDpiForMonitor` returns an HRESULT, not a pointer

Reply:
Good catch. Based on http://www.pinvoke.net/default.aspx/Enums/HRESULT.html, I'll use uint instead of IntPtr